cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,106 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
   it('', () => {
+    cy.get('[id="firstName"]')
+      .type('Tom');
```

Comment:
All data should be randomized with the use of faker. Use can achieve it by adding a new command to the support folder or config file.